src/sth_helper.js

```diff
@@ -0,0 +1,161 @@
+/* globals module, process */
+
+(function() {
+  "use strict";
+
+  var sthConfig, sthLogger;
+
+  /**
+   * Stops the server asynchronously
+   * @param {hapi.Server} server The server to be stopped
+   * @param {Function} callback Callback function to notify the result
+   *  of the operation
+   */
+  function stopServer(server, callback) {
```

Comment:
You're absolutely right ;) In fact I created https://github.com/telefonicaid/IoT-STH/issues/5 for this but since you are raising the issue now, I'll move the function now and I'll close https://github.com/telefonicaid/IoT-STH/issues/5 Thanks! ;)